code/modules/awaymissions/corpse.dm

```diff
@@ -750,6 +750,34 @@
 	corpsemask = /obj/item/clothing/mask/breath
 	corpsehelmet = /obj/item/clothing/head/helmet/space/capspace
 
+/obj/effect/landmark/corpse/waifu
+	name = "Waifu"
+	corpseuniform = /obj/item/clothing/under/schoolgirl
+	corpseshoes = /obj/item/clothing/shoes/kneesocks
+	corpsehelmet = /obj/item/clothing/head/kitty
+	corpseradio = /obj/item/device/radio/headset
+	corpseid = 1
+	corpseidjob = "Waifu"
+	corpseidaccess = "Assistant"
+	corpsegender = G_FEMALE
+
+/obj/effect/landmark/corpse/waifu/secfu
+	name = "Arie"
```

Comment:
delet

Reply:
We actually established a precedent years ago in the Gashapon PR to not have shit with staticnames on it, that's why we have a Toy Snowflake instead of a Toy Hime

Reply:
Oof. The meme went too far. It'll be taken out.